Browser not launched through Cypress problem

### Current behavior:
When running any test using Chrome, the browser opens and shows the following message: **This browser was not launched through Cypress. Tests cannot run.**, and a *Run Chrome 64* button appears as well. Clicking the button launches the same browser again with the same message.

### Desired behavior:
The test should be executed properly when running them on Chrome.

### How to reproduce:

1. Open cypress: `cypress open`
2. Select a test in the user interface and run it (Chrome selected as the browser)
Notice that the tests don't get executed

#### Test code:
Not relevant to the scenario. Same behaviour happen when running cypress examples.

### Additional Info (images, stack traces, etc)
At the moment, in order to workaround this issue, I close the browser, run the test using Electron, the close it down and run again using Chrome. This way the test runs properly.

This happens when starting a new instance of cypress. After having it fixed through the workaround, the tests run properly regardless of if we close the browser or not.

I uninstalled Chrome and deleted all profile information, but the issue kept happening.

Note this issue is only happening on one machine. Any help to investigate would be appreciated. 

- Operating System: macOS High Sierra. Version 10.13.2 (17C88)
- Cypress Version: 1.4.1
- Browser Version: Google Chrome Version 64.0.3282.119 (Official Build) (64-bit)

Hi,

I would like to giveyou some additional information about my setup. 
By run npm install cypress, I get the results:

> cypress@2.1.0 postinstall C:\Mesapplications\Repository\node_modules\cypress
> node index.js --exec install

Installing Cypress (version: 2.1.0)

 ×  Downloading Cypress
   → Cypress Version: 2.1.0
    Unzipping Cypress
    Finishing Installation
The Cypress App could not be downloaded.

Please check network connectivity and try again:
----------

URL: https://download.cypress.io/desktop/2.1.0?platform=win32&arch=x64
Error: tunneling socket could not be established, statusCode=407
----------

Platform: win32 (10.0.14393)
Cypress Version: 2.1.0
npm WARN enoent ENOENT: no such file or directory, open 'C:\Mesapplications\Repository\package.json'
npm WARN Repository No description
npm WARN Repository No repository field.
npm WARN Repository No README data
npm WARN Repository No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cypress@2.1.0 postinstall: `node index.js --exec install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cypress@2.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

I use a proxy and the download of Cypress is always stopped. I founded another solution by downloading directly the .zip file (https://cdn.cypress.io/desktop/2.0.4/win64/cypress.zip) on another PC (not using this proxy). When I opened Cypress, I always got "Whoops, we can't run yours tests" and it is true that I can't display "google.com" in Cypress GUI...I think finally that my issue is linked to my proxy. What do you think about that?

Regards,
Elisa

--- a/packages/extension/app/manifest.json
+++ b/packages/extension/app/manifest.json
@@ -8,6 +8,7 @@
     "https://*/*",
     "<all_urls>"
   ],
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAugoxpSqfoblTYUGvyXZpmBgjYQUY9k2Hx3PaDwquyaTH6GBxitwVMSu5sZuDYgPHpGYoF4ol6A4PZHhd6JvfuUDS9ZrxTW0XzP+dSS9AwmJo3uLuP88zBs4mhpje1+WE5NGM0pTzyCXYTPoyzyPRmToALWD96cahSGuhG8bSmaBw3py+16qNKm8SOlANbUvHtEaTpmrSWBUIq7YV8SIPLtR8G47vjqPTE1yEsBQ3GAgllhi0cJolwk/629fRLr3KVckICmU6spXD/jVhIgAeyHhFuFGYNuubzbel8trBVw5Q/HE5F6j66sBvEvW64tH4lPxnM5JPv0qie5wouPiT0wIDAQAB",
   "icons": {
     "16":  "icons/icon_16x16.png",
     "48":  "icons/icon_48x48.png",

--- a/packages/extension/test/unit/extension_spec.coffee
+++ b/packages/extension/test/unit/extension_spec.coffee
@@ -1,5 +1,6 @@
 require("../spec_helper")
 
+exec      = require("child_process").exec
 fs        = require("fs-extra")
 path      = require("path")
 Promise   = require("bluebird")
@@ -8,6 +9,7 @@ extension = require("../../index")
 cwd       = process.cwd()
 
 fs = Promise.promisifyAll(fs)
+exec = Promise.promisify(exec)
 
 describe "Extension", ->
   context ".getCookieUrl", ->
@@ -87,3 +89,13 @@ describe "Extension", ->
           fs.readFileAsync(@src, "utf8")
         .then (str2) ->
           expect(str).to.eq(str2)
+
+  context "manifest", ->
+    it "has a key that resolves to the static extension ID", ->
+      fs.readJsonAsync(path.join(cwd, "app/manifest.json"))
+      .then (manifest) ->
+        cmd = "echo \"#{manifest.key}\" | openssl base64 -d -A | shasum -a 256 | head -c32 | tr 0-9a-f a-p"
+        exec(cmd)
+        .then (stdout) ->
+          expect(stdout).to.eq("caljajdfkjjjdehjdoimjkkakekklcck")
+

